nanome/_internal/_plugin.py

```diff
@@ -43,6 +43,7 @@ def __parse_args(self):
                 Logs.message(" -r, --auto-reload    restart plugin automatically if a .py or .json file in current directory changes")
                 Logs.message(" --ignore             to use with auto-reload. All paths matching this pattern will be ignored, " \
                     "use commas to specify several. Supports */?/[seq]/[!seq]")
+                Logs.message(" -nl                  disables .log file")
```

Comment:
Can we rename it `--no-log-file`? It's a few more characters, but its more descriptive

Reply:
I'm down to have both. Feel free to directly change the branch 😃

Reply:
Not a fan of multiple characters for a single dash, it feels wrong to me since other cli tools allow you to combine single character flags (e.g. `ls -la`). `--no-log-file` is fine but I really don't like `-nl`